packages/flutter/lib/src/animation/ticker.dart

```diff
@@ -74,6 +74,6 @@ class Ticker {
   void _scheduleTick() {
     assert(isTicking);
     assert(_animationId == null);
-    _animationId = scheduler.requestAnimationFrame(_tick);
+    _animationId = scheduler.scheduleFrameCallback(_tick);
```

Comment:
:)
I have another more experimental branch where I return Subscription objects.
Currently the post and persistent callbacks can't even be canceled.

I will see, if I can integrate the Subscription changes already in this PR.